### ISSUE TITLE
1.3.2 missing file in archive

### ISSUE THREAD

**Issue by remicollet** [flaupretre/pecl-pcs#8]
```
In file included from /dev/shm/BUILD/php56-php-pecl-pcs-1.3.2/NTS/php_pcs.c:38:0:
/dev/shm/BUILD/php56-php-pecl-pcs-1.3.2/NTS/pecl-compat/compat.h:107:31: fatal error: src/zend_resource.h: No such file or directory
 #include "src/zend_resource.h"
                               ^

```

File exists on github but is not listed in package.xml, so missing in pecl archive.

**Comment by remicollet** [comment]
File added in aa14e4395a6447283b86b92864d9de5c6a5bb2b1

**Comment by flaupretre** [comment]
Just released PCS 1.3.3. This version includes this PR and a minor change fixing a compile warning causing travis failure. Hope we're stable now :)
